climada/entity/exposures/base.py

```diff
@@ -906,15 +906,29 @@ def copy(self, deep=True):
             **metadata
         )
 
-    def write_raster(self, file_name, value_name='value', scheduler=None):
+    def write_raster(self, file_name, value_name=['value'], scheduler=None):
         """Write value data into raster file with GeoTiff format
 
         Parameters
         ----------
         file_name : str
-            name output file in tif format
+            output tif file to write to
+        value_name : str or list of str
+            column(s) in the geodataframe containing data. If multiple values are provided, each column will be
+            written to a separate band in the output raster
+        scheduler : str, optional
+            used for dask map_partitions.
+            “threads”, “synchronous” or “processes”
         """
-        if self.meta and self.meta['height'] * self.meta['width'] == len(self.gdf):
+        if not isinstance(value_name, list):
+            value_name = [value_name]
+
+        # Simple case: one raster band, every grid value defined in the Exposures.
+        # Then we can skip a slow points_to_raster call:
+        if self.meta and \
+                'height' in self.meta.keys() and \
+                len(value_name) == 1 and \
+                self.meta['height'] * self.meta['width'] == len(self.gdf):
```

Comment:
This has a weird and hard-to-read indention.

Reply:
Can you please still fix this indention?

Also, in line with the other pull request #343:

There is now a big repetition of the lines 
```        if self.meta and \
                'height' in self.meta.keys() and \
                len(value_name) == 1 and \
                self.meta['height'] * self.meta['width'] == len(self.gdf)
```
                
 This is ugly, and can lead to issues in the future. It would be better to
 a) fix the issue with inconsistent meta data at its roots
b) simply add a function to the exposures that makes this check. 

Thus, this can be removed from here, and be maintenained easily, and be read easily without the need for cryptic and quickly obsolete in-line comments.

Reply:
yeah i am in favour of `Exposures.is_raster` and `Hazard.is_raster` methods

Reply:
I think that's easier than reviewing/rewriting everything `meta`, for now.

If you're ok with that I _won't_ fix this indentation and will replace it with an `is_raster` check in another pull request.

Reply:
I think this is good path forward. Could you also open an issue about the general handling of  `meta` ?

Reply:
Can do